gstreamer-editing-services/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2017 Sebastian Dröge <sebastian@centricular.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate libc;
+
+use std::sync::{Once, ONCE_INIT};
+
+extern crate glib_sys as glib_ffi;
+extern crate gobject_sys as gobject_ffi;
+extern crate gio_sys as gio_ffi;
+extern crate gstreamer as gst;
+extern crate gstreamer_sys as gst_ffi;
+extern crate gstreamer_base as gst_base;
+extern crate gstreamer_base_sys as gst_base_ffi;
+extern crate gstreamer_pbutils_sys as gst_pbutils_ffi;
+extern crate gstreamer_pbutils as gst_pbutils;
+extern crate gstreamer_editing_services_sys as ffi;
+
+use glib::translate::{from_glib};
+
+#[macro_use]
+extern crate glib;
+extern crate gio;
+
+static GES_INIT: Once = ONCE_INIT;
+
+pub fn init() -> Result<(), &'static str> {
+    unsafe {
+        if from_glib(ffi::ges_init()) {
```

Comment:
You might want to also first call `gst::init()` here?

Reply:
I am not sure, do you think it is better?

Reply:
Probably not. But is explicit initialization of GES really needed or can we just do it lazily?